platform/ui/src/assets/styles/styles.css

```diff
@@ -1,7 +1,9 @@
 /* CUSTOM OHIF SCROLLBAR */
 .ohif-scrollbar {
-  scrollbar-color: #173239 transparent;
+  scrollbar-color: #041c4a transparent;
```

Comment:
are we changing the scrollbar color all together in the app?

Reply:
I don't think the current scrollbar color fits with anything at the moment, that's why I decided to update it all together, if not a good idea then I can make it specific to the new changes.